.pronto.yml

```diff
@@ -0,0 +1,15 @@
+all:
+  exclude:
+    - 'spec/**/*'
+# exclude files for single runner
+eslint:
+  exclude:
+    - 'app/assets/**/*'
+github:
+  slug: prontolabs/pronto
+  access_token: B26354
+  api_endpoint: https://api.github.com/
+  web_endpoint: https://github.com/
+max_warnings: 150
+warnings_per_review: 30
+verbose: false
```

Comment:
Add a new line at the end of the file. Github displays an icon at the last line to highlight the issue.
Read more info in https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
It is possible that you can configure your editor to add the empty line automatically. Check the editor setting for more info.

Reply:
Fixed in last commit